test-support/helpers/interactions.js

```diff
@@ -8,7 +8,7 @@ import {
 
 const { isEmpty } = Ember;
 
-export function checkByLabel(labelText) {
+export function clickByLabel(labelText) {
```

Comment:
@Daniel-Xu I think `clickRadioByLabel()` might be redundant at that point, because we're actually clicking on the radio input via the label: https://github.com/DockYard/ember-cli-test-interactions/blob/af182185bd8a5fe34bb021f96a9807efa44c8e11/test-support/helpers/interactions.js#L40